lib/foreman_tasks/dynflow/console_authorizer.rb

```diff
@@ -3,7 +3,8 @@ module ForemanTasks
   class Dynflow::ConsoleAuthorizer
     def initialize(env)
       @rack_request          = Rack::Request.new(env)
-      @user_id, @expires_at = @rack_request.session.values_at('user', 'expires_at')
+      @user_id = @rack_request.session[:user]
```

Comment:
please drop the alignment of assignment on other rows of the method (or align all)